core/src/main/java/org/hibernate/ogm/jpa/impl/OgmJpaStoredProcedureQuery.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.jpa.impl;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.StoredProcedureQuery;
+
+import org.hibernate.HibernateException;
+import org.hibernate.QueryParameterException;
+import org.hibernate.jpa.internal.StoredProcedureQueryImpl;
+import org.hibernate.jpa.spi.AbstractEntityManagerImpl;
+import org.hibernate.jpa.spi.HibernateEntityManagerImplementor;
+import org.hibernate.ogm.storedprocedure.impl.NoSQLProcedureCallImpl;
+import org.hibernate.ogm.storedprocedure.impl.NoSQLProcedureOutputImpl;
+import org.hibernate.ogm.util.impl.Log;
+import org.hibernate.ogm.util.impl.LoggerFactory;
+import org.hibernate.procedure.ParameterRegistration;
+import org.hibernate.procedure.ProcedureCallMemento;
+
+/**
+ * @author Sergey Chernolyas &amp;sergey_chernolyas@gmail.com&amp;
+ */
+public class OgmJpaStoredProcedureQuery extends StoredProcedureQueryImpl {
```

Comment:
I would remove the Jpa from the name, it's already in the package

Reply:
ok

Reply:
fixed